heimdall/CMakeLists.txt

```diff
@@ -15,6 +18,8 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
 
 if(MINGW)
     set(CMAKE_EXE_LINKER_FLAGS "-static-libgcc -static-libstdc++ -static")
+else(NOT MINGW)
+	set(CMAKE_EXE_LINKER_FLAGS "-lobjc -framework IOKit -framework CoreFoundation")
```

Comment:
Using the changes from [e.foundation](https://gitlab.e.foundation/e/tools/heimdall/-/commit/af5c83a45c8580c3c264db008cd0582c0cf1b757) is cleaner than the changes suggested here, since they don't break compilation for other platforms. This works and compiles well on [my fork](https://github.com/amo13/Heimdall):